src/app/components/server-charts/server-charts.component.ts

```
@@ -0,0 +1,791 @@
+
+    this.geekbenchHTML += `</ul>`;
+
+    this.geekbenchHTML = this.sanitizer.bypassSecurityTrustHtml(this.geekbenchHTML);
+
+    if(dataSet && dataSet.length) {
+      let labels: string[] = [];
+      let scales: string[] = [];
+
+      const geekBenchScore = dataSet?.find(x => (x.benchmark_id as string).includes('geekbench:score'));
+
+      if(geekBenchScore && geekBenchScore.benchmarks.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)